docs/mkdocs.yml

```diff
@@ -476,6 +476,7 @@ nav:
         - listitem list: cmd/spo/listitem/listitem-list.md
         - listitem set: cmd/spo/listitem/listitem-set.md
         - listitem remove: cmd/spo/listitem/listitem-remove.md
+        - listitem retentionlabel remove: cmd/spo/listitem/listitem-retentionlabel-remove.md
```

Comment:
This line should be sorted alphabetically with the subcommands of listitem. Hence it should come after `listitem batch add`